pallets/xcmp-handler/src/mock.rs

```diff
@@ -284,7 +292,7 @@ impl pallet_xcmp_handler::Config for Test {
 	type GetNativeCurrencyId = GetNativeCurrencyId;
 	type SelfParaId = parachain_info::Pallet<Test>;
 	type AccountIdToMultiLocation = AccountIdToMultiLocation;
-	type CurrencyIdToMultiLocation = ();
+	type CurrencyIdToMultiLocation = TokenIdConvert;
```

Comment:
Is this `CurrencyIdToMultiLocation` tested?

In other words, is there a test passing with () before and with TokenIdConvert now, or there’s none for it.

Reply:
Good question. Before the changes in this PR the CurrencyIdToMultiLocation was not being adequately tested by the existing tests. This pr changed that by generating the xcm skeleton for fees. With the new fees code we need to use are real mock for TokenIdConvert.

If this change is removed, this is the failing test:
```
---- tests::calculate_xcm_fee_doesnt_charge_for_alternate_flow stdout ----
thread 'tests::calculate_xcm_fee_doesnt_charge_for_alternate_flow' panicked at 'assertion failed: `(left == right)`
  left: `Err(Module(ModuleError { index: 3, error: [9, 0, 0, 0], message: Some("CurrencyNotSupported") }))`,
 right: `Ok((0, 700000000))`', pallets/xcmp-handler/src/tests.rs:228:9
```

Reply:
Cool, so the conversion is covered now.

Reply:
yup 👍